src/sentry/workflow_engine/endpoints/serializers.py

```diff
@@ -126,11 +128,30 @@ def get_attrs(
             for group, serialized in zip(condition_groups, serialize(condition_groups, user=user))
         }
 
+        filtered_item_list = [item for item in item_list if item.type == ErrorGroupType.slug]
+        project_ids = [item.project_id for item in filtered_item_list]
```

Comment:
the ids are guaranteed to be unique b/c 1 error detector per project right?

Reply:
yes, but that is a good point.